packages/mobile/src/components/spinner/spinner.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { FC } from 'react';
+import { ActivityIndicator, View } from 'react-native';
+import { styles } from './styles';
+
+type Props = {
+  size?: number;
+  color?: string;
+  isOverflow?: boolean;
+};
+
+const Spinner: FC<Props> = ({ size, color, isOverflow }) => {
+  if (isOverflow) {
+    return (
+      <View style={styles.overflowContainer}>
+        <ActivityIndicator size={size ?? 45} color={color} />
```

Comment:
Let's extract default spinner size to constant and use the same value for both variants (overflow true/false)